frontend/pages/beta/settings.tsx

```diff
@@ -0,0 +1,62 @@
+import Box from '@mui/material/Box'
+import Tab from '@mui/material/Tab'
+import Tabs from '@mui/material/Tabs'
+import React, { useState } from 'react'
+
+import { useGetCurrentUser } from '../../actions/user'
+import SettingsProfileTab from '../../src/settings/beta/SettingsProfileTab'
+import ThemeTab from '../../src/settings/beta/ThemeTab'
+import Wrapper from '../../src/Wrapper.beta'
+
+function TabPanel({ children, value, index, ...rest }) {
+  return (
+    <Box
+      role='tabpanel'
+      hidden={value !== index}
+      id={`simple-tabpanel-${index}`}
+      aria-labelledby={`simple-tab-${index}`}
+      {...rest}
+    >
+      {value === index && <Box sx={{ p: 3 }}>{children}</Box>}
```

Comment:
We should conditionally render each `TabPanel` using this logic, rather than hiding the box and conditionally rendering the content. Also the second `Box` here is unecessary as the `p: 3` could be added to the wrapping `Box` above.